src/backend/routers/user.py

```diff
@@ -35,20 +40,22 @@ async def check_password(password: str, hash_: str) -> bool:
 
 async def set_session(
     user_profile: UserProfile, sessions: AbstractSessionStorage, message: str
-) -> dict[str, T]:
```

Comment:
We can now remove [this TypeVar](https://github.com/idkbrowby/gamingrealm-backend/blob/db91a311f136d748992d22ce104ff334febbfa27/src/backend/routers/user.py#L21) since it's unused.

Reply:
Resolved in [d7c3fe2](https://github.com/idkbrowby/gamingrealm-backend/pull/34/commits/d7c3fe237123e277b01b5c81e1f2a26fd4b1586b)